neurodamus/gap_junction.py

```diff
@@ -98,6 +103,10 @@ def configure_connections(self, conn_conf):
 
     def finalize(self, *_, **_kw):
         super().finalize(conn_type="Gap-Junctions")
+        if self.cell_manager.population_name == \
+                SimConfig.beta_features.get("gapjunction_target_population"):
```

Comment:
Just another style nitpick 🙈 
```suggestion
        gj_target_pop = SimConfig.beta_features.get("gapjunction_target_population")
        if self.cell_manager.population_name == gj_target_pop:
```

Reply:
Thanks, done!